src/styles/product-card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 166px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  color: $main;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__descreption {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+
+    &-data {
+      font-size: 10px;
+      font-weight: 400;
+      margin-bottom: 1px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 15px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      color: $secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 600;
+    }
+  }
+
+  &__button {
+    height: 38px;
+    border-radius: 5px;
+    border: 1px solid $light-blue;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    color: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
Thanks, done!